app/helpers/era_av_helper.rb

```diff
@@ -0,0 +1,29 @@
+# Copyright 2017, University of Alberta Libraries.
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+#   under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+#   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+#   specific language governing permissions and limitations under the License.
+# ---  END LICENSE_HEADER BLOCK  ---
+
+module EraAvHelper
+  def ualib_logo_link
+    link_to("", "https://library.ualberta.ca", :class => 'logo')
+  end
+
+  def era_av_logo_link
```

Comment:
Something I pushed back on chris a long time ago. 

But why not always have the logo as a link to the root_path? Why complicate this and change it's behaviour depending if your on the homepage or elsewhere on the site. It's extra logic that goes against what every site on the planet does (google, facebook, github, you name it, etc. all have their logos link back to the homepage, even if they are already on the homepage).

Then this helper isn't needed either and this whole file can get deleted.